apps/src/applab/designMode.js

```diff
@@ -506,6 +507,11 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
     var msg = "The following lines of HTML were modified or removed:\n" + removed +
       "\noriginal html:\n" + unsafe + "\nmodified html:\n" + safe;
     console.log(msg);
+    logToCloud.addPageAction(logToCloud.PageAction.SanitizedLevelHtml, {
+      removedHtml: removed.substring(0, logToCloud.MAX_LENGTH),
+      unsafeHtml: unsafe.substring(0, logToCloud.MAX_LENGTH),
+      safeHtml: safe.substring(0, logToCloud.MAX_LENGTH)
+    });
```

Comment:
I wonder if we should have addPageAction just iterate through each of the fields, and truncate them if need be.